wasi-runner/local_setup.sql

```diff
@@ -1,7 +1,4 @@
 DROP TABLE IF EXISTS bots;
-DROP USER IF EXISTS snippyuser;
-
-CREATE USER snippyuser WITH PASSWORD 'snippy123';
```

Comment:
This might disrupt some of your flow but with the containerized version of postgres the user gets created via env variables.

Reply:
I have a mild preference to keep snippyuser and the root postgres user separate from each other (so that the permissions for snippyuser can be kept to a minimum), but honestly, either way is fine. :)

Reply:
Good point, adopted your suggestion.